src/tlo/methods/rti.py

```diff
@@ -3429,6 +3438,13 @@ def apply(self, person_id, squeeze_factor):
         # determine the number of ICU days used to treat patient
 
         if df.loc[person_id, 'rt_ISS_score'] > self.hdu_cut_off_iss_score:
+
+            # TODO: some general ICU equipment listed below? additional would need to be added dependent on severity
+            #  of illness
```

Comment:
Will this be kept here for future improvement, is it? You don't want to deal with this within this PR, correct?

Reply:
This is hard to say - I dont think there is much ICU equipment in equipment file as it stands. Maybe something to discuss during clinical review